app/Http/Controllers/SubmissionsController.php

```diff
@@ -2,32 +2,31 @@
 
 namespace App\Http\Controllers;
 
+use App\Conference;
+use App\Submission;
 use Illuminate\Http\Request;
-use App\Commands\CreateSubmission;
-use App\Commands\DestroySubmission;
 
 class SubmissionsController extends Controller
 {
     public function store(Request $request)
     {
-        $talk = auth()->user()->talks()->findOrFail($request->input('talkId'));
-
-        $this->dispatch(new CreateSubmission(
-            $request->input('conferenceId'),
-            $talk->id
-        ));
-
-        return response()->json(['status' => 'success', 'message' => 'Talk Submitted']);
+        $talk = auth()->user()->talks()->findOrFail($request->input('talkId'))->current();
+        $conference = Conference::findOrFail($request->input('conferenceId'));
+
+        $submission = Submission::create([
```

Comment:
Can we do:

```php
$conference->submissions()->create(['talk_revision_id' => $talk->id]);
```

?

Also, if we're passing a talk_revision ID, is this not the wrong ID?